test.py

```diff
@@ -15,9 +18,25 @@ def test_sample(self):
   #   else:
   #     return 0
 
+  # def test_input_less_than_random(self) :
+  #   self.assertEqual(chat_food(2,4), -1)
+
 
```

Comment:
What's going on here:
Each test (test_chat_Activities_coding, test_chat_Activities_hiking) uses a different mocked function for the `get_input` function (see main.py line 5 for this definition!).

Basically our `test_chat_Activities_coding` test is saying "okay Python, when you **want** to call that `get_input()` function, call this function `mocked_get_input_coding` instead!"
mocked_get_input_coding is going to return the string 'coding' which is basically the same as if your user typed that as their response!